bundles/statistics/statsgrid2016/components/IndicatorParametersList.js

```diff
@@ -114,10 +104,12 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParametersList', funct
         }
         return id;
     },
+    resetMyIndicatorTable: function () {
+        this.getElement().find('.my-indicator').empty();
+    },
     resetIndicatorSelectors: function (showInsertButton) {
```

Comment:
The parameter isn't used anymore since you removed the row that set the button visibility. Remove the param.